library/tests/src/com/google/maps/android/data/kml/KmlPolygonTest.java

```diff
@@ -5,6 +5,7 @@
 import junit.framework.TestCase;
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class KmlPolygonTest extends TestCase {
```

Comment:
I believe these changes to `KmlPolygonTest` are unrelated to adding the ability to return the index for the closest point on the line.  If so, can you please remove these changes from this changeset?